packages/Algernon.package/ALGAction.class/instance/do.st

```diff
@@ -0,0 +1,3 @@
+accessing
+do
+	self theTarget perform: self theSelector withArguments: self theArgs asArray
```

Comment:
Is there a way to remove metaprogramming here?

Reply:
Hm, it's not that simple. The Types create an multiple Actions to perform code, which is found type classes. Therefore a bit of Metaprogramming is needed at least for the Actions and Names. We could move the "Action-Generation2 Part from the Types to Action Class Methods, but I'm not sure if that would really improve the current situation. 

If you have a different/better idea, feel free to share it with me. 🙂

Reply:
Could we just pass a block?

Reply:
So that each action has a block associated with it. I think that would make a lot of code nicer. Especially in places where actions need multiple arguments as this method only supports one argument.

Reply:
Several arguments are also possible now. (As array). 
With blocks I worry that the "Debug-ability" would suffer, because you cannot look into blocks so easily.

Reply:
Is that a problem with blocks? I have not experienced that.

Reply:
Yes we are, but therefore we can't easily view the Arguments or the executed selector in the debugger (Since both are "embedded" in the block).
Although I have just learned that blocks at least have the method "sourceString", so it is at least possible with one "intermediate" step 🤔.

Reply:
But I accept your judgement as this is your pr.